dbtc/client/cloud/base.py

```diff
@@ -77,10 +59,28 @@ def _header_property(self):
 
         return 'api_key'
 
+    def _clone_resource(self, resource: str, **kwargs):
```

Comment:
@matt-winkler A private method that we can use for other resources we'd like to clone